.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,74 @@
+name: Nightly integration tests
+
+# Run on request and every day at 3 AM UTC
+on:
+  workflow_dispatch:
+    inputs:
+      syntax_check:
+        type: boolean
+        required: true
+        description: 'Run syntax checker.'
+      target:
+        description: 'Target'
+        required: true
+        default: 'none'
+        type: choice
+        options:
+          - none
+          - quantinuum
+      target_machine:
+        type: string
+        required: false
+        description: 'Target machine (e.g., H1-1E).'
```

Comment:
You need to set a default here (should be the syntax checker). Otherwise the workflow below will fail if the target_machine is not specified.